src/libs/AxelarQueryAPI.ts

```diff
@@ -162,14 +162,16 @@ export class AxelarQueryAPI {
    * @param sourceChainTokenSymbol
    * @param gasLimit (Optional) An estimated gas amount required to execute `executeWithToken` function. The default value is 700000 which should be sufficient for most transactions.
    * @param gasMultiplier (Optional) A multiplier used to create a buffer above the calculated gas fee, to account for potential slippage throughout tx execution, e.g. 1.1 = 10% buffer. supports up to 3 decimal places
+   * @param minGasFee (Optional) A floor set for the gas price in wei, used as override in case calculated gas fee is below desired minimum
```

Comment:
`used as override in case calculated gas fee is below desired minimum`

I think this isn't what the Squid team wants. `minGasPrice` should be compared with the `gas_price` that returned from the `getNativeGasBaseFee` function (`response.sourceToken.gas_price`), then multiply it by the `gasLimit`. 

For example, 

Given the following params:
- gasLimit = 1e7
- minGasPrice = 500 gwei

and the `getNativeGasBaseFee` API returns gas_price = '30 gwei'

Then, `destTxFee` should be Math.max(minGasPrice, gas_price) * gasLimit

Then, follows the current flow to calculate the final fee.

Reply:
gotcha! you're right. updated PR accordingly